### ISSUE TITLE
Get rid of excess amount of ping log in the controller

### ISSUE THREAD

**Issue by yasker** [rancher/longhorn#121]
Logs like this floods the real useful logs.

```
::1 - - [02/Jun/2016:20:51:59 +0000] "GET /v1/volumes HTTP/1.1" 200 686
::1 - - [02/Jun/2016:20:51:59 +0000] "GET /v1/volumes HTTP/1.1" 200 686
10.42.101.144 - - [02/Jun/2016:20:51:59 +0000] "GET /v1/replicas HTTP/1.1" 200 588
10.42.101.144 - - [02/Jun/2016:20:52:00 +0000] "GET /v1/replicas HTTP/1.1" 200 588
::1 - - [02/Jun/2016:20:52:01 +0000] "GET /v1/replicas HTTP/1.1" 200 584
::1 - - [02/Jun/2016:20:52:01 +0000] "GET /v1/replicas HTTP/1.1" 200 584
::1 - - [02/Jun/2016:20:52:01 +0000] "GET /v1/replicas HTTP/1.1" 200 584
10.42.175.66 - - [02/Jun/2016:20:52:01 +0000] "GET /v1/replicas HTTP/1.1" 200 588
10.42.175.66 - - [02/Jun/2016:20:52:04 +0000] "GET /v1/replicas HTTP/1.1" 200 588
::1 - - [02/Jun/2016:20:52:04 +0000] "GET /v1/volumes HTTP/1.1" 200 686
::1 - - [02/Jun/2016:20:52:04 +0000] "GET /v1/volumes HTTP/1.1" 200 686
10.42.101.144 - - [02/Jun/2016:20:52:05 +0000] "GET /v1/replicas HTTP/1.1" 200 588
10.42.101.144 - - [02/Jun/2016:20:52:05 +0000] "GET /v1/replicas HTTP/1.1" 200 588
::1 - - [02/Jun/2016:20:52:06 +0000] "GET /v1/replicas HTTP/1.1" 200 584
::1 - - [02/Jun/2016:20:52:06 +0000] "GET /v1/replicas HTTP/1.1" 200 584
::1 - - [02/Jun/2016:20:52:06 +0000] "GET /v1/replicas HTTP/1.1" 200 584
```

**Comment by kp6** [comment]
@yasker I don't seem to find where those logs are edited from... Do you happen to know by any chance?

**Comment by yasker** [comment]
No, I haven't looked into them myself. Can be in the dependency package like go-rancher etc. @ibuildthecloud may knows more.

**Comment by yasker** [comment]
@cjellick do you know where these logs come from?

**Comment by kp6** [comment]
ok, I figured how this logging works: 
- The controller installs a 3rd party Apache format logger into the http server handler chain
- Therefore all http requests are logged after being processed

**Comment by kp6** [comment]
@yasker 
Questions:
- Do you suggest we filter out from the logs all http requests with `/v1/replicas` in the url?
- Or some more sophisticated filtering?

**Comment by yasker** [comment]
No.

It's specify to the log shown above, which is:

```
::1 - - [02/Jun/2016:20:51:59 +0000] "GET /v1/volumes HTTP/1.1" 200 686
10.42.101.144 - - [02/Jun/2016:20:51:59 +0000] "GET /v1/replicas HTTP/1.1" 200 588
```

**Comment by kp6** [comment]
In other words `GET` requests for:
- `/v1/volumes` 
- `/v1/replicas`

**Comment by yasker** [comment]
Yeah.

**Comment by yasker** [comment]
Fixed by #183
